schemas/adaptive-card.json

```diff
@@ -1014,25 +1321,132 @@
 					]
 				},
 				"weight": {
+					"$ref": "#/definitions/TextWeight"
+				},
+				"wrap": {
+					"$ref": "#/definitions/Wrap"
+				}
+			},
+			"required": [
+				"type",
+				"text"
+			]
+		},
+		"TextRun": {
+			"additionalProperties": true,
+			"type": "object",
+			"description": "Defines a single run of formatted text",
+			"version": "1.2",
+			"properties": {
+				"type": {
 					"type": "string",
-					"description": "Controls the weight of `TextBlock` elements.",
+					"description": "Must be `\"TextRun\"`.",
 					"enum": [
-						"lighter",
-						"default",
-						"bolder"
+						"TextRun"
 					]
 				},
-				"wrap": {
+				"color": {
+					"$ref": "#/definitions/TextColor"
+				},
+				"fontType": {
+					"$ref": "#/definitions/FontType"
+				},
+				"horizontalAlignment": {
+					"$ref": "#/definitions/HorizontalAlignment"
+				},
```

Comment:
Oh one last catch - horizontalAlignment isn't part of TextRun (since it's an inline, there's no concept of horizontal alignment)